src/integrationTest/java/uk/gov/hmcts/darts/usermanagement/controller/SecurityRoleControllerIntTest.java

```diff
@@ -0,0 +1,107 @@
+package uk.gov.hmcts.darts.usermanagement.controller;
+
+import org.junit.jupiter.api.Test;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
+import uk.gov.hmcts.darts.authorisation.component.UserIdentity;
+import uk.gov.hmcts.darts.testutils.IntegrationBase;
+import uk.gov.hmcts.darts.testutils.stubs.AdminUserStub;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@AutoConfigureMockMvc
+class SecurityRoleControllerIntTest extends IntegrationBase {
+
+    public static final String ENDPOINT_URL = "/admin/security-roles";
+
+    @Autowired
+    private AdminUserStub adminUserStub;
+
+    @MockBean
+    private UserIdentity userIdentity;
+
+    @Autowired
+    private transient MockMvc mockMvc;
+
+    @Test
+    void createSecurityGroupShouldSucceedWhenProvidedWithValidValuesForMinRequiredFields() throws Exception {
+        adminUserStub.givenUserIsAuthorised(userIdentity);
+
+        MockHttpServletRequestBuilder requestBuilder = get(ENDPOINT_URL)
+            .contentType(MediaType.APPLICATION_JSON_VALUE);
+
+        MvcResult mvcResult = mockMvc.perform(requestBuilder).andExpect(status().isOk()).andReturn();
+
+        String actualJson = mvcResult.getResponse().getContentAsString();
+        String expectedJson = """
```

Comment:
is there a standard/recommendation for response strings in tests - a lot of the integration tests have them defined in an expectedResponse.json file?

Reply:
no standard, but there was a discussion last week on darts-modernisation, one person does suggest using there strings instead of response files:-
https://hmcts-reform.slack.com/archives/C04T0CVJ28N/p1705662974658419?thread_ts=1705658275.075549&cid=C04T0CVJ28N